### PR TITLE
Writing_Native_Bears.rst: Typo error fixed

### PR DESCRIPTION
**Pull request by S2606**
Fixes `pip` to `pip3` for better readability.

Closes https://github.com/coala/coala/issues/4413

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by S2606** [comment]
@IpshitaC  on setting head commit message as `pip3 inconcistency fixed`, I am getting the following error,

`Shortlog of HEAD commit does not match given regex: ([^:]*|[^:]+[^ ]: [A-Z0-9*].*)`

Can you tell me what should i do?

**Comment by S2606** [comment]
@prachi1210 As i shall change more than one file, so what should i write in header message?
Should i mention all the file names which would be edited??I am bit confused.

**Comment by prachi1210** [comment]
If you are changing more than one file, you can omit the name of the file from the shortlog and just have a commit message saying
```
Replaces pip with pip3

Updates all occurrences of pip to pip3
Closes https://github.com/coala/coala/issues/4413
```

**Comment by nkprince007** [comment]
All the changes look OK. But a retrigger of build is necessary to fix the broken Circle CI tests.

**Comment by S2606** [comment]
@nkprince007 How can i retrigger build as circle ci isnt allowing me from their interface since i dont have write access??

**Comment by supergr35** [comment]
Just amend your commit with no changes and push once again, and your builds will restart.

**Comment by nkprince007** [comment]
ack 
https://github.com/coala/coala/pull/4450/commits/e4a7abe61d8f6d9a4a620039bd0c9f6e6e911014

**Comment by RJ722** [comment]
@S2606 can you please change the commit message to something like:
```
Replace pip with pip3
Update all occurrences of pip to pip3

```
It needs to be in imperative mood 😉

**Comment by Asnelchristian** [comment]
@S2606 address @RJ722  issue, rebase and :tada: :+:

**Comment by RJ722** [comment]
ack https://github.com/coala/coala/pull/4450/commits/c13cfed26ae6fd0730ab31d386028c250271e536

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -4,28 +4,28 @@ coala 0.10 - PolarBear
 ::
```
Comment (RJ722) [inline RELEASE_NOTES.rst]: @S2606 You should also change the message here:
* https://github.com/coala/coala/blob/master/coalib/collecting/Collectors.py#L141
* https://github.com/coala/coala/blob/master/coalib/misc/Constants.py#L24

**Comment by RJ722** [comment]
unack https://github.com/coala/coala/commit/c13cfed26ae6fd0730ab31d386028c250271e536

File: RELEASE_NOTES.rst (hunk 1 above)
Comment (S2606) [inline RELEASE_NOTES.rst]: @RJ722 Updated Changes.

Comment (RJ722) [inline RELEASE_NOTES.rst]: You need to make changes in the unittests as well, that's why the CI tests are failing.

Comment (RJ722) [inline RELEASE_NOTES.rst]: Here: https://github.com/coala/coala/blob/master/tests/coalaTest.py

Comment (S2606) [inline RELEASE_NOTES.rst]: @RJ722  Updated changes.

**Comment by adhikasp** [comment]
ack 59af404c74379a2fd015448bb67719ec5757a016

**Comment by Nosferatul** [comment]
@S2606 please rebase

**Comment by gureks** [comment]
ack 86f0544

**Pull request merged**